google/resource_compute_disk_test.go

```diff
@@ -200,6 +200,35 @@ func testAccCheckComputeDiskDestroy(s *terraform.State) error {
 	return nil
 }
 
+func testAccCheckComputeDiskExistsInProject(n, p string, disk *compute.Disk) resource.TestCheckFunc {
```

Comment:
I don't like that I had to do this (this is a copy of testAccCheckComputeDiskExists with an extra project parameter, which I hate) but I wasn't sure how I could avoid that.  The other method (here and in all three files) uses the default project.  I'm a little too new to Go to know what the right move here was - I wanted to make the project an argument in testAccCheckComputeDiskExists, but don't know how to get the name of the default project at the call site.  I assume this cannot be right - please let me know what _would_ be right.

Reply:
`getTestProjectFromEnv()` is the method you are looking for.

Reply:
Ahh, thank you!  That seems better.  I'll make that change and ping you again.